client/app/app.scss

```diff
@@ -58,3 +58,8 @@ body,
     width: 100%;
   }
 }
+
+// Makes the outline on focus less intrusive.
+:focus {
+  outline: thin dotted !important;
+}
```

Comment:
What do you think about removing this all together? I feel like our UI is very intuitive as is, maybe down the road we can remove this style on elements that don't necessarily need to be highlighted when focused.